docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,358 @@
+|--|--|--|--|--|--|--|
+| | | | | | | |
+
+### Stiching Together
+
+
+Business Glossary will be a first class entity where one can define the business terms and this will be similar to entities like Dataset, CorporateUser etc.
+Then extending the Dataset's aspect SchemaMetadata, that will be extending the SchemaField model to capture urn of the business term (optional).
+
+![high level design](business_glossary_rel.png)
+
+The above diagram illustrates how business glossaries will be connected to other entities. Business terms are modelled as BusinessTerm entities and example business terms are `Term-1`, `Term-2`, .. `Term-n`. In the above diagram from the dataset (`DS-1`) element `e11` is linked to business term `Term-2` and `e12` is linked to `Term-1`. At the same time from Dataset (`DS-2`) element `e24` linked the business term `Term-2`, `e22` with `Term-3` and `e24` with `Term-4`.
+
+## Metadata Model Enhancements 
+
+There will be 1 top level GMA [entities](../../../what/entity.md) in the design: businessTerm (Business Glossary).
+It's important to make businessTerm as a top level entity because it can exist without a Dataset and can be defined independently by the business team.
+
+### URN Representation
+We'll define a [URNs](../../../what/urn.md): `BusinessTermUrn`.
+These URNs should allow for unique identification of business term.  
+
+A business term  URN (BusinessTermUrn) will look like below:
+```
+urn:li:businessTerm:(<<namespace>>,<<name>>)
+```
+
+A Dataset Field URN(DatasetFieldUrn will be like below (this is being added as part of another RFC)
+```
+urn:li:datasetField:(<datasetUrn>,<fieldPath>)
+```
+
+### New Snapshot Object
+There will be new snapshot object to onboard business terms along with definitions
+
+Path : metadata-models/src/main/pegasus/com/linkedin/metadata/snapshot/
+```java
+/**
+ * A metadata snapshot for a specific BusinessTerm entity.
+ */
+record BusinessTermSnapshot {
+
+  /**
+   * URN for the entity the metadata snapshot is associated with.
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * The list of metadata aspects associated with the dataset. Depending on the use case, this can either be all, or a selection, of supported aspects.
+   */
+  aspects: array[BusinessTermAspect]
+}
+```
+
+Path : metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/
+
+### BusinessTermAspect
+There will be new aspect defined to capture the required attributes & ownership information
+
+```
+/**
+ * A union of all supported metadata aspects for a BusinessTerm
+ */
+typeref BusinessTermAspect = union[
+  BusinessTermProperties,
+  Ownership
+]
+```
+
+Business Term Entity Definition
+```java
+/**
+ * Data model for a Business Term entity
+ */
+record BusinessTermEntity includes BaseEntity {
+
+  /**
+   * Urn for the dataset
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * Business Term native name e.g. CashInstrument
+   */
+  name: optional string
+
+}
+```
+
+### Entity BusinessTermInfo
+
```

Comment:
Didn't get this .. why is this entity and where is this model used ?

Reply:
this is the primary object to hold the Business term definition/properties (earlier I had named this as `BusinessTermProperies` and renamed to `BusinessTermInfo` in the last iteration. Do let e know your thoughts